Mathlib/NumberTheory/RamificationInertia/Galois.lean

```diff
@@ -207,4 +211,70 @@ theorem ncard_primesOver_mul_ramificationIdxIn_mul_inertiaDegIn [IsGalois K L] :
 
 end fundamental_identity
 
+section inertia
+
+variable {R S G : Type*} [CommRing R] [CommRing S] [Algebra R S] [Group G]
+  [MulSemiringAction G S] [SMulCommClass G R S] [Algebra.IsInvariant R S G] [Finite G]
+
+open scoped Pointwise
+
+open Algebra
+
+attribute [local instance 1001] Ideal.Quotient.field Module.Free.of_divisionRing in
+lemma ncard_primesOver_mul_card_inertia_mul_finrank (p : Ideal R) [p.IsMaximal]
+    (P : Ideal S) [P.LiesOver p] [P.IsMaximal] [Algebra.IsSeparable (R ⧸ p) (S ⧸ P)] :
+    (p.primesOver S).ncard * Nat.card (P.toAddSubgroup.inertia G) *
+      Module.finrank (R ⧸ p) (S ⧸ P) = Nat.card G := by
+  classical
+  trans (p.primesOver S).ncard * Nat.card (MulAction.stabilizer G P); swap
+  · rw [← IsInvariant.orbit_eq_primesOver R S G p P]
+    simpa using Nat.card_congr (MulAction.orbitProdStabilizerEquivGroup G P)
+  rw [mul_assoc]
+  cases nonempty_fintype G
+  have : IsGalois (R ⧸ p) (S ⧸ P) := { __ := Ideal.Quotient.normal (A := R) G p P }
+  have := Ideal.Quotient.finite_of_isInvariant G p P
+  congr 1
+  have : Subgroup.index _ = _ := Nat.card_congr
+    (QuotientGroup.quotientKerEquivOfSurjective (Ideal.Quotient.stabilizerHom P p G)
+      (Ideal.Quotient.stabilizerHom_surjective G p P)).toEquiv
+  rw [← IsGalois.card_aut_eq_finrank, ← this]
+  convert (Ideal.Quotient.stabilizerHom P p G).ker.card_mul_index using 2
+  have : (Ideal.Quotient.stabilizerHom P p G).ker = (P.toAddSubgroup.inertia G).subgroupOf _ := by
```

Comment:
Can you make this a separate lemma and can you add a formulation like this which is slightly stronger:
```
theorem Ideal.inertiaSubgroup_eq_ker : P.toAddSubgroup.inertia G =
    (Ideal.Quotient.stabilizerHom P p G).ker.map (MulAction.stabilizer G P).subtype
```
I've got a proof in #27053, but there's probably a cleaner proof.

Reply:
@erdOne can you please do this? Otherwise LGTM.